gemfiles/Gemfile.common.rb

```diff
@@ -8,10 +8,6 @@
   gem 'simplecov', :require => false
 end
 
-platforms :rbx do
```

Comment:
RBX has been dropped at https://github.com/mongomapper/mongomapper/commit/48c31ca6053d9536378ae69f728e9ea32309b03f, and rubysl seems not to be compatible with ruby 3.0.